osu.Game/Overlays/OverlayView.cs

```diff
@@ -15,18 +21,52 @@ namespace osu.Game.Overlays
     /// Automatically performs a data fetch on load.
     /// </remarks>
     /// <typeparam name="T">The type of the API response.</typeparam>
-    public abstract class OverlayView<T> : CompositeDrawable, IOnlineComponent
+    public abstract class OverlayView<T> : Container, IOnlineComponent
         where T : class
     {
         [Resolved]
         protected IAPIProvider API { get; private set; }
 
+        private LoadingSpinner spinner { get; }
+
+        private const double transform_time = 600;
+
+        protected override Container<Drawable> Content { get; } = new Container
+        {
+            RelativeSizeAxes = Axes.X,
+            AutoSizeAxes = Axes.Y,
+        };
+
+        private readonly Placeholder currentPlaceholder;
+
+        private readonly BlockingBox blockingBox;
```

Comment:
All these members read *really* messy in this order. Please group them by access modifier and type as per [this ordering](https://github.com/ppy/osu-framework/issues/2278).